src/zzz_od/gui/app_setting/coffee_setting_interface.py

```
@@ -41,29 +46,46 @@ def __init__(self, ctx: ZContext, parent=None):
 
+        self.challenge_group = ExpandSettingCardGroup(icon=FluentIcon.GAME, title='喝后挑战', content=CoffeeChallengeWay.ALL.value.desc)
+        content_widget.add_widget(self.challenge_group)
+
         self.challenge_way_opt = ComboBoxSettingCard(icon=FluentIcon.GAME, title='喝后挑战', options_enum=CoffeeChallengeWay)
-        content_widget.add_widget(self.challenge_way_opt)
+        self.challenge_way_opt.value_changed.connect(self.on_challenge_way_changed)
+        self.challenge_group.addHeaderWidget(self.challenge_way_opt.combo_box)
 
-        self.card_num_opt = ComboBoxSettingCard(icon=FluentIcon.GAME, title='体力计划外的数量', options_enum=CoffeeCardNumEnum)
-        content_widget.add_widget(self.card_num_opt)
+        self.card_num_opt = ComboBoxSettingCard(icon=FluentIcon.GAME, title='体力计划外的数量', content='喝后挑战找不到对应体力计划配置时，使用这里设置的卡片数量', options_enum=CoffeeCardNumEnum)
```

Ruff (0.15.21)

[warning] String contains ambiguous `，` (FULLWIDTH COMMA). Did you mean `,` (COMMA)?

(RUF001)